docs/pages/reference/backends.mdx

```diff
@@ -651,6 +651,10 @@ Service reads these parameters to configure its interactions with S3:
 - `use_fips_endpoint=true` -  [Configure S3 FIPS
   endpoints](#configuring-aws-fips-endpoints)
 
+- `use_s3_path_style` - Whether to use path-style instead of virtual-host-style URLs for the
```

Comment:
Defaulting to "true" is usually tricky because it's hard to make the difference between "false" and "unset", you have to use a bool pointer which I see you're using below. Why not invert this to `use_virtual_style_addressing` (or something along those lines) that defaults to "false", wouldn't it make things simpler?